srcopsmetrics/utils.py

```diff
@@ -25,7 +25,53 @@
 from typing import Tuple, Dict, Any, List, Optional
 from pathlib import Path
 
+from github import Github
+import time
+from datetime import datetime
+
 _LOGGER = logging.getLogger(__name__)
+API_RATE_MINIMAL_REMAINING = 20
+
+class knowledge:
```

Comment:
What about upper first letter for Classes? :)

Reply:
oh yeah! I don't know why I thought that context managers can be lower case :D